### PR TITLE
Update basic-tutorials.md

### PR DESCRIPTION
**Pull request by logankilpatrick**
Updating the initial getting started section with a little more detail and a link to the Getting Help section.

### CODE REVIEW

**Comment by logankilpatrick** [comment]
Related to #122

File: docs/src/basic-tutorials.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Note that before you put anything in the scene, it will be blank!
 
 ## Getting help
 
-The user-facing functions of Makie are pretty well documented, so you can usually use the help mode in the REPL, or your editor of choice.
+The user-facing functions of Makie are pretty well documented, so you can usually use the help mode in the REPL, or your editor of choice. If you countinue to have issues, see [Getting Help](http://makie.juliaplots.org/dev/index.html#Getting-Help:-1).
```
[the remark this answers is not in the document]
Comment (asinghvi17) [reply docs/src/basic-tutorials.md]: wait what just happened

File: docs/src/basic-tutorials.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Note that before you put anything in the scene, it will be blank!
 
 ## Getting help
 
-The user-facing functions of Makie are pretty well documented, so you can usually use the help mode in the REPL, or your editor of choice.
+The user-facing functions of Makie are pretty well documented, so you can usually use the help mode in the REPL, or your editor of choice. If you countinue to have issues, see [Getting Help](@tbreloff ).
```
Comment (asinghvi17) [inline docs/src/basic-tutorials.md]:
```suggestion
The user-facing functions of Makie are pretty well documented, so you can usually use the help mode in the REPL, or your editor of choice. If you countinue to have issues, see [Getting Help](@ref).

Comment (asinghvi17) [reply docs/src/basic-tutorials.md]: Sorry, it looks like autocorrect struck again.  I was wondering why there was a mention of the `@tbreloff` macro...

File: docs/src/basic-tutorials.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (logankilpatrick) [reply docs/src/basic-tutorials.md]: you suggested that, and I committed it

[the remark this answers is not in the document]
Comment (asinghvi17) [reply docs/src/basic-tutorials.md]: true...sorry about that...didn't see autocorrect.  Latest suggestion should have fixed that.

**Comment by logankilpatrick** [comment]
Should be all set!

**Comment by asinghvi17** [comment]
Yup, thanks!

**Pull request merged**